src/chrome/content/dicts.js

```diff
@@ -144,7 +144,7 @@ euskalbar.dicts = function () {
 
         mimeType: dict.mimeType || '',
 
-        data: dict.getParams(term, source, target),
+        data: dict.getParams(term, source, target, true),
```

Comment:
Please don't fall into the [boolean trap](http://ariya.ofilabs.com/2011/08/hall-of-api-shame-boolean-trap.html). Consider reverting.